backends/platform/androidsdl/androidsdl-sdl.cpp

```diff
@@ -42,7 +44,101 @@ void OSystem_ANDROIDSDL::initBackend() {
 
 	if (!ConfMan.hasKey("gfx_mode"))
 		ConfMan.set("gfx_mode", "2x");
+	
+	if (!ConfMan.hasKey("swap_menu_and_back"))
+		ConfMan.setBool("swap_menu_and_back", true);
+	else
+		swapMenuAndBackButtons(ConfMan.getBool("swap_menu_and_back"));
+	
+	if (!ConfMan.hasKey("touchpad_mouse_mode")) {
+		const bool enable = (SDL_ANDROID_GetMouseEmulationMode() == 0) ? false : true;
+		ConfMan.setBool("touchpad_mouse_mode", enable);
+	} else
+		touchpadMode(ConfMan.getBool("touchpad_mouse_mode"));
+	
+	if (!ConfMan.hasKey("onscreen_control")) {
+		const bool enable = (SDL_ANDROID_GetScreenKeyboardShown() == 0) ? false : true;
```

Comment:
You can just write this like:

    const bool enable = SDL_ANDROID_GetScreenKeyboardShown();